libs/components/src/lib/tag/tag.spec.ts

```diff
@@ -0,0 +1,311 @@
+		});
+	});
+
+	describe('connotation', () => {
+		it('sets correct internal connotation style', async () => {
+			const connotation = 'cta';
+			(element as any).connotation = connotation;
+			await elementUpdated(element);
+
+			const base = element.shadowRoot?.querySelector(`.base.connotation-${connotation}`);
+			expect(base)
+				.toBeInstanceOf(Element);
+		});
+	});
+
+	describe('shape', () => {
+		it('sets correct internal shape style', async () => {
+			const shape = 'pill';
+			(element as any).shape = shape;
+			await elementUpdated(element);
+
+			const base = element.shadowRoot?.querySelector(`.base.shape-${shape}`);
+			expect(base)
+				.toBeInstanceOf(Element);
+		});
+	});
+
+	describe('appearance', () => {
+		it('sets correct internal appearance style', async () => {
+			const appearance = 'duotone';
+			(element as any).appearance = appearance;
+			await elementUpdated(element);
+
+			const base = element.shadowRoot?.querySelector(`.base.appearance-${appearance}`);
+			expect(base)
+				.toBeInstanceOf(Element);
+		});
+	});
+
+	describe('disabled', () => {
+		it('should set disabled class when disabled is true', async () => {
+
+			element.toggleAttribute('disabled', true);
+			await elementUpdated(element);
+
+			const base = element.shadowRoot?.querySelector('.base.disabled');
+			expect(base).toBeInstanceOf(Element);
+		});
+	});
+
+	describe('selectable', () => {
+		it('should init to false', async () => {
+			expect(element.selectable)
+				.toEqual(false);
+			expect(element.hasAttribute('selectable'))
+				.toEqual(false);
+		});
+
+		it('should toggle attribute on host', async () => {
+			await toggleSelectable(element);
+			const removeAttributeExistsWhenTrue = element.hasAttribute('selectable');
+
+			await toggleSelectable(element, false);
+			const removeAttributeExistsWhenFalse = element.hasAttribute('selectable');
+
+			expect(removeAttributeExistsWhenTrue)
+				.toEqual(true);
+			expect(removeAttributeExistsWhenFalse)
+				.toEqual(false);
+		});
+
+		it('should set selectable property on attribute change', async () => {
+			element.toggleAttribute('selectable');
+			await elementUpdated(element);
+			expect(element.selectable)
+				.toEqual(true);
+		});
+
+		it('should remove the selectable icon when selectable is false', async () => {
+			expect(element.shadowRoot?.querySelector('.selectable-icon'))
+				.toEqual(null);
+		});
+
+		it('should remove the selectable icon on click', async () => {
+			await toggleSelectable(element, true);
+			element.click();
+			await elementUpdated(element);
+			expect(element.shadowRoot?.querySelector('.selectable-icon'))
+				.toEqual(null);
+		});
+
+		it('should update selected to true when selectable', async () => {
+			await toggleSelectable(element, true);
+			getBaseElement(element).click();
+			await elementUpdated(element);
+			expect(element.selected).toBeTruthy();
+		});
+
+		it('should not update selected to true when not selectable', async () => {
```

Comment:
Better to be "positive" when writing test descriptions :)
```suggestion
		it('should leave selected unchanged when not selectable', async () => {
```